### DISCUSSION TITLE
Error running pod install

### DISCUSSION THREAD

**Discussion by afijal** [codemagic-ci-cd/codemagic-docs#1289] [Q&A] [answered]
Out of nowhere we started getting error when running pod install, locally it works fine.

` -> Installing GoogleMLKit (2.2.0)
     > Http download
       $ /usr/bin/curl -f -L -o /var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-t8uchj/file.tgz https://dl.google.com/dl/cpdc/4ed6ad0451716ebe/GoogleMLKit-2.2.0.tar.gz --create-dirs --netrc-optional --retry 2 -A 'CocoaPods/1.11.3 cocoapods-downloader/1.6.3'

    [!] Error installing GoogleMLKit
    [!] /usr/bin/curl -f -L -o /var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-t8uchj/file.tgz https://dl.google.com/dl/cpdc/4ed6ad0451716ebe/GoogleMLKit-2.2.0.tar.gz --create-dirs --netrc-optional --retry 2 -A 'CocoaPods/1.11.3 cocoapods-downloader/1.6.3'

    curl: (26) .netrc parser error

    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader.rb:144:in `rescue in execute_command'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader.rb:141:in `execute_command'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-downloader-1.6.3/lib/cocoapods-downloader/base.rb:175:in `block in executable'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-downloader-1.6.3/lib/cocoapods-downloader/http.rb:22:in `download_file'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-downloader-1.6.3/lib/cocoapods-downloader/remote_file.rb:25:in `download!'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-downloader-1.6.3/lib/cocoapods-downloader/base.rb:83:in `block in download'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader.rb:159:in `block in ui_action'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:64:in `section'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader.rb:158:in `ui_action'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-downloader-1.6.3/lib/cocoapods-downloader/base.rb:81:in `download'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader.rb:110:in `download_source'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader.rb:77:in `download_request'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader/cache.rb:256:in `download'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader/cache.rb:239:in `block in uncached_pod'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader/cache.rb:266:in `in_tmpdir'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader/cache.rb:238:in `uncached_pod'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader/cache.rb:33:in `download_pod'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/downloader.rb:42:in `download'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/pod_source_installer.rb:118:in `download_source'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer/pod_source_installer.rb:69:in `install!'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:563:in `install_source_of_pod'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:514:in `block (2 levels) in install_pod_sources'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:86:in `titled_section'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:513:in `block in install_pod_sources'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:496:in `each'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:496:in `install_pod_sources'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:255:in `block in download_dependencies'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/user_interface.rb:64:in `section'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:254:in `download_dependencies'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/installer.rb:162:in `install!'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/command/install.rb:52:in `run'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/claide-1.1.0/lib/claide/command.rb:334:in `run'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/lib/cocoapods/command.rb:52:in `run'
    /Users/builder/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/cocoapods-1.11.3/bin/pod:55:in `<top (required)>'
    /Users/builder/.rbenv/versions/3.0.4/bin/pod:25:in `load'
    /Users/builder/.rbenv/versions/3.0.4/bin/pod:25:in `<main>'

Error output from CocoaPods:
↳
         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-q7mhhy'...
         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-d1hdgs'...
         Note: switching to '78424be314842833c04bc3bef5b72e85fff99204'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-1yxab6'...
         Note: switching to 'a727e44718a67e300089174e5591166045815ba4'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-n45jn'...
         Note: switching to 'edd41d8a4e2d563b14628dd4a1acf8230314d9ae'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-nmwhaz'...
         Note: switching to '04ed759b8a0d1bd1f326f4589a25a5080731b6f1'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-x0vbih'...
         Note: switching to '7397e456a68ee344737bf377b1b76ba4fdd7a1fa'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-dutuiv'...
         Note: switching to 'c24031ad9410c746c49deddc739fdf311a386fc7'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-zm5xs9'...
         Note: switching to 'c24031ad9410c746c49deddc739fdf311a386fc7'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-x6nlpw'...
         Note: switching to 'dce2e1abc6c0d5e830ff1cffe3f8633fda64001e'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-7h78zf'...
         Note: switching to 'dce2e1abc6c0d5e830ff1cffe3f8633fda64001e'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-f33isj'...
         Note: switching to 'c24031ad9410c746c49deddc739fdf311a386fc7'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-3zfgur'...
         Note: switching to '4e9bbf2808b8fee444e84a48f5f3c12641987d3e'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

         Cloning into '/var/folders/w2/rrf5p87d1bbfyphxc7jdnyvh0000gn/T/d20230104-2508-lytz46'...
         Note: switching to '5056b15c5acbb90cd214fe4d6138bdf5a740e5a8'.

         You are in 'detached HEAD' state. You can look around, make experimental
         changes and commit them, and you can discard any commits you make in this
         state without impacting any branches by switching back to a branch.

         If you want to create a new branch to retain commits you create, you may
         do so (now or later) by using -c with the switch command. Example:

           git switch -c <new-branch-name>

         Or undo this operation with:

           git switch -

         Turn off this advice by setting config variable advice.detachedHead to false

       curl: (26) .netrc parser error

Error running pod install`

Any idea? Other pods before this one download just fine

**Comment by himesh-cm** [comment]
@afijal, Can you please share the build id?

**Reply by afijal** [reply]
@himesh-cm 
Sure
63b5aea86cdd500253f5bfc4
63b57362c1cad8a11e848110
I can have more if needed

**Reply by himesh-cm** [reply]
Thanks, i will check and get back

**Reply by himesh-cm** [reply]
I am taking this with the devs. Meanwhile, can we switch to Xcode 14.1 instead of the latest 14.2 for debugging purposes and re-run the builds to see if it makes any difference?

**Reply by afijal** [reply]
@himesh-cm let me check it

**Reply by afijal** [reply]
@himesh-cm runa  few builds and it seems like 14.1 works fine, thanks!

**Reply by himesh-cm** [reply]
Do monitor the same, and let us know if it resurfaces as i understand the issue is intermittent.

**Reply by afijal** [reply]
@himesh-cm Sure, and please let me know if you find out the reason and we can try xcode 14.2 again

**Reply by benja8151** [reply]
Same issue here, build id: 63bbf0e1c3f11bc5daa103c3 . Works fine with XCode 14.1

**Reply by himesh-cm** [reply]
Hi @benja8151, yes we have identified the issue and narrowed it down as below.
```
iOS issue persists randomly on 14.2 only when we have a single iOS build without Android on M1.
```

**Reply by atnpem** [reply]
Hello, I am experiencing a similar issue where it only occurs on specific workflows but not others. It is weird as it is the same code and configurations, just a different flavor. Run with 14.1 fix this issue :
* 63d04850d12ece2085a2cce0 -> build success with 14.1
* 63d0423c796ad3f1d1bd367a -> fail with 14.2
* 63d03e84ba825c66a608df9e -> fail with 14.2
* 63d03aeaba825c66a608deea -> fail with 14.2

But I have success on 14.2 with the same code but a different flavor :
* 63d031f11f541724754058ff
* 63d03c67802af2a1b85ff880

Tell us when we can set up the last Xcode version again.

Thank you

**Reply by himesh-cm** [reply]
Hi @antoinepemeja, Devs are already looking into it.
You can change the `Xcode version` within the `Build Settings` of your workflow editor.

**Comment by himesh-cm** [comment, marked as answer]
Xcode 14.1 can be used as a workaround.

**Reply by woroud2000** [reply]
Same problem... but Xcode 14.1 **didn't** change any thing.
PS: Neither 14.0 / 14.2 / 14.3. Flutter 3.3 / 3.7 / 3.10.
All of them failed ;(

**Comment by jeiea** [comment]
It seems there's still the problem.

**Reply by himesh-cm** [reply]
@jeiea, Can you please share the build id, so we can look into it.

**Reply by jeiea** [reply]
It's 63e0a77db1b8ff6e38ba8ea5. I didn't downgrade xcode in the build.

**Comment by nataddrho** [comment]
https://github.com/orgs/codemagic-ci-cd/discussions/1645
